lib/Migration/Version0019Date20200603080001.php

```diff
@@ -427,10 +427,15 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			);
 			$table->addColumn(
 				'cached_name', 'string', [
-							 'notnull' => false,
-							 'length'  => 255,
-							 'default' => ''
-						 ]
+								 'notnull' => false,
+								 'length'  => 255,
+								 'default' => ''
+							 ]
+			);
+			$table->addColumn(
+				'cached_update', 'datetime', [
+								   'notnull' => false,
+							   ]
```

Comment:
Add a new migration instead of changing existing ones. Otherwise upgrades will do 💥

Reply:
there is a new migration also (Version0019Date20201106110741), no worry !